docs/adr/0010-transition-fxa-from-backbone-to-react.md

```diff
@@ -0,0 +1,151 @@
+
+### Option D - Move to Angular
+
+- Description
+  - Use AngularJS.
+- Pros
+  - Angular is backed by a well-known company (Google) with a lot of dedicated resources working on the library and it's popular, has a lot of community support, and has good documentation.
+  - It provides some features we likely would use in FxA without another package dependency. Certain things may just "work" automagically.
+- Cons
+  - Angular has a steep learning curve.
+  - Since Angular is a full-on "framework," it results in a heavier bundle and can result in a slower application compared to React or Vue.
+  - It provides many features we likely wouldn't use in FxA.
+  - Angular is opinionated and expects things to be done in specific ways. While this could make it easier for a developer already familiar with Angular to ramp up on the project, this could set limitations on how we want to architect our application.
+  - Significant changes have historically been introduced to the framework.
+
+### Option E - Move to VueJS
+
+- Pros:
+  - Vue is used proudly by the community of developers that support it. The enthusiasm for open-source and attitude around making something "for developers" aligns with the Mozilla mission.
+  - Vue is flexible - it allows app structure as seen fit and supports many different HTML templating languages.
+  - Likely has the smallest learning curve.
+- Neutral:
+  - Allows for mutable state.
+- Cons:
+  - Vue is the newest library of the bunch, has a much smaller community than other options, and does not have a large company backing. It may not be as battle-tested as other options as a result or have as much documentation support when we run into specific issues.
+
+### Option F - Other considerations
+
+Web components were also considered. However, these are not a library, but are instead "a set of web platform APIs that allow you to create new custom, reusable, encapsulated HTML tags to use in web pages and web apps." They offer strong encapsulation and can be used in any JS framework or library that works with HTML, but don't meet the needs of FxA by themselves. See a more detailed description [with examples here](https://css-tricks.com/an-introduction-to-web-components/).
+
+## Links
+
+- [Create a new React app for Settings Redesign](https://github.com/mozilla/fxa/blob/master/docs/adr/0011-create-new-react-app-for-settings-redesign.md)
+- Other Backbone pros/cons: [subscription management stack ADR](https://github.com/mozilla/fxa/blob/master/docs/adr/0002-use-react-redux-and-typescript-for-subscription-management-pages.md)
+- [React](https://github.com/facebook/react)
+- [NextJS](https://github.com/zeit/next.js/)
+- [AngularJS](https://github.com/angular/angular.js/)
+- [VueJS](https://github.com/vuejs/vue)
+- [Web Components](https://css-tricks.com/an-introduction-to-web-components/)
```

Comment:
These links aren't being used in the doc ^

Reply:
Other ADRs have "additional info" links at the bottom for links that aren't used in the doc. I'm tempted to leave these as-is, but I could remove them or add them where mentioned within the ADR.